docs/CONTRIBUTING.md

```diff
@@ -46,14 +47,44 @@ Thank you for your interest in contributing to Nishiki! This document outlines t
 - Make sure all tests pass locally and on the CI server.
 - Update existing tests if you modify code that affects them.
 
+## Commit Convention
+
+Before you create a Pull Request, please check whether your commits comply with
+the commit conventions used in this repository.
+
+When you create a commit we kindly ask you to follow the convention
+`category(scope or module): message` in your commit message while using one of
+the following categories:
+
+- `feat / feature`: all changes that introduce completely new code or new
```

Comment:
Does this mean I should commit a feature by saying either `feat {commit message}` or `feature {commit message}`?
Or, `feat / feature {commit message}`?

Reply:
It means commit messages must be prefixed with either `feat:` or `feature:`.
I will update it by removing `feat:`, so it would be clearer.

Reply:
#20